### DISCUSSION TITLE
Passing flags to `chezmoi git`

### DISCUSSION THREAD

**Discussion by Archie-2021** [twpayne/chezmoi#4004] [Q&A]
I want to run `chezmoi git status -s` but it don't pass the flags to git, I even tried `--` in the command but didn't work. Am I doing something wrong or it is a bug?

**Comment by twpayne** [comment]
Run

```console
$ chezmoi git -- status -s
```

**Reply by Archie-2021** [reply]
> Run
> 
> ```
> $ chezmoi git -- status -s
> ```

oh sorry
I think i put the -- before git and that is why it didn't work in the first try
Thanks
